scripts/database-credentials.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+port="3306";
+ip="127.0.0.1";
+name="bassa";
+read -p "Database type: " DBTYPE;
+read -p "Database port [$port]: " DBPORT;
+read -p "Database ip [$ip]: " DBIP;
+read -p "Database name [$name]: " DBNAME;
+read -p "Database username: " DBUSERNAME;
+read -p "Database password: " DBPASSWORD;
```

Comment:
@padamchopra  I think we have to make the password hidden, like, when the user types the password on the terminal, it should not show the password typing. 

See: https://stackoverflow.com/a/3980904/1190184